# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1325886556 18000
#      Fri Jan 06 16:49:16 2012 -0500
# Branch stable
# Node ID 906eecd4bf43b0146c50019ee41acac2dc83c935
# Parent  7d6b8bc3fa2f4c35ea2f1aace003b4938b9f955d
directly link eigs.oct with ARPACK libs if needed

* src/DLD-FUNCTIONS/module-files (eigs.cc): Include ARPACK flag and
library variables.

diff --git a/src/DLD-FUNCTIONS/module-files b/src/DLD-FUNCTIONS/module-files
--- a/src/DLD-FUNCTIONS/module-files
+++ b/src/DLD-FUNCTIONS/module-files
@@ -27,17 +27,17 @@ convhulln.cc|$(QHULL_CPPFLAGS)|$(QHULL_L
 daspk.cc
 dasrt.cc
 dassl.cc
 det.cc
 dlmread.cc
 dmperm.cc|$(SPARSE_XCPPFLAGS)|$(SPARSE_XLDFLAGS)|$(SPARSE_XLIBS)
 dot.cc
 eig.cc
-eigs.cc|$(SPARSE_XCPPFLAGS)|$(SPARSE_XLDFLAGS)|$(SPARSE_XLIBS) $(LAPACK_LIBS) $(BLAS_LIBS)
+eigs.cc|$(ARPACK_CPPFLAGS) $(SPARSE_XCPPFLAGS)|$(ARPACK_LDFLAGS) $(SPARSE_XLDFLAGS)|$(ARPACK_LIBS) $(SPARSE_XLIBS) $(LAPACK_LIBS) $(BLAS_LIBS)
 fft.cc|$(FFTW_XCPPFLAGS)|$(FFTW_XLDFLAGS)|$(FFTW_XLIBS)
 fft2.cc|$(FFTW_XCPPFLAGS)|$(FFTW_XLDFLAGS)|$(FFTW_XLIBS)
 fftn.cc|$(FFTW_XCPPFLAGS)|$(FFTW_XLDFLAGS)|$(FFTW_XLIBS)
 fftw.cc|$(FFTW_XCPPFLAGS)|$(FFTW_XLDFLAGS)|$(FFTW_XLIBS)
 filter.cc
 find.cc
 gammainc.cc
 gcd.cc
